demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -221,26 +240,47 @@ def _get_all_packages(content_dir: str) -> List[str]:
 
     @staticmethod
     def _filter_changed_packages(content_repo: git.Repo, pkgs: List[Path], base_branch: str) -> List[Path]:
-        """ Checks which packages had changes using git (working tree, index, diff between HEAD and master in them and should
-        run on Lint.
+        """ Checks which packages had changes in them and should run on Lint.
```

Comment:
what is the effort to convert this function logic to use the git_utils?

Reply:
Discusses in person